demos/3d_spherical/3d_spherical.py

```diff
@@ -29,6 +29,7 @@
 
 mesh2d = CubedSphereMesh(rmin, refinement_level=ref_level, degree=2)
 mesh = ExtrudedMesh(mesh2d, layers=nlayers, extrusion_type='radial')
+mesh.cartesian = False
```

Comment:
Same comment as `2d_cylindrical`: maybe this should be explained in the cell above.

Reply:
See previous comment. I would also suggest in the base case, where this flag is specified for the first time, that we flag it is a requirement and why.

Reply:
Perhaps saying something like:
      cartesian:
        - True: gravity is assumed to point in the negative z-direction
        - False: gravity is assumed to point radially inward